src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,4 @@
+// review
```

Comment:
There's a couple of things here you don't have to do - the first is wrapping the full return value in parentheses - you don't often see this in ternaries. The other is enforcing `boolean` to be a Boolean - because you are returning a string, you only need to be concerned with the truthiness/falsiness of what's passed in.